Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,1249 @@
+
+
+def issue_query_command(client: JiraBaseClient, args: Dict[str, str]) -> List[CommandResults] | CommandResults:
+    jql_query = args.get('query', '')
+    start_at = arg_to_number(args.get('start_at', ''))
+    max_results = arg_to_number(args.get('max_results', ''))
+    headers = args.get('headers')
+    specific_fields = argToList(args.get('specific_fields', ''))
+    query_specific_fields: List[str] = []
+    if specific_fields:
+        specific_fields = set(field.lower() for field in specific_fields)
+    custom_id_name_mapping, custom_fields_res = get_custom_fields_mapping(client=client)
+    specific_fields_mapping: Dict[str, str] = {}
+    for field_key, field_value in custom_id_name_mapping.items():
+        if(field_value.lower() in specific_fields):
+            specific_fields_mapping[field_key] = field_value
+            query_specific_fields.append(field_key)
+    res = client.run_query(jql=jql_query, start_at=start_at, max_results=max_results, specific_fields=query_specific_fields)
+    if not res:
+        return CommandResults(readable_output='No issues matched the query.')
+    command_results: List[CommandResults] = []
+    for issue in res.get('issues', []):
+        markdown_dict: Dict[str, Any] = {}
+        outputs: Dict[str, Any] = {}
+        for specific_field_key, specific_field_value in specific_fields_mapping.items():
+            titled_field = specific_field_value.title().replace(' ', '')
+            markdown_dict[titled_field] = outputs[titled_field] = demisto.get(issue, f'fields.{specific_field_key}', '')
+            markdown_dict['Id'] = outputs['Id'] = demisto.get(issue, 'id', '')
+            markdown_dict['Key'] = outputs['Key'] = demisto.get(issue, 'key', '')
+        if not (markdown_dict or outputs):
+            # If the markdown and outputs dictionaries are empty, that means the specific fields given do not exist,
+            # in such a case, we return the values that are returned when creating an issue.
+            markdown_dict, outputs = create_issue_md_and_outputs(client=client, res=issue)
+        command_results.append(
+            CommandResults(
+                outputs_prefix='Ticket',
+                outputs=outputs,
+                outputs_key_field='Id',
+                readable_output=tableToMarkdown(name=f'Issue {outputs.get("Key", "")}', t=markdown_dict,
+                                                headers=argToList(headers),
+                                                headerTransform=pascalToSpace),
+                raw_response=issue
+            ),
+        )
+    return command_results
+
+
+def get_comments_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    issue_id_or_key = args.get('issue_id', '') if args.get('issue_id', '') else args.get('issue_key', '')
+    if not issue_id_or_key:
+        return_error(ID_OR_KEY_MISSING_ERROR)
+    limit = arg_to_number(args.get('limit', None))
+    if not limit:
+        limit = 50
```

Comment:
This won't really help me since `arg_to_number` returns (int | None), therefore, I always need to check if it is None, and if it is, then give the default value, since I want limit to be an integer, not (int | None).